src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/DataSource.java

```diff
@@ -22,7 +25,12 @@
 @Builder
 @Entity(name = "DataSource")
 @Table(name = "data_source")
-public class DataSource extends BaseEntity {
+public class DataSource extends AuditEntity {
```

Comment:
mmm, what about a situation where there is more than one administrator that can add data sources from a UI? I think it may be necessary to know who performed the actions on the data sources

Reply:
The whole entity currently contains a lot of unneeded information/attributes but okay let's leave and see.